lib/templates/deploy.bash.ruby.template

```diff
@@ -1,6 +1,12 @@
 # Rails Helpers
 # -----------
 
+setLatestRubyVersion(){
+    # Hardcoded to 2.3.3 for now until we support more versions
+	LATEST_RUBY_VERSION=2.3.3
```

Comment:
Seems like something to clean up sooner rather than later. See https://github.com/projectkudu/kudu/blob/ce36b37c255ff87538ababc086f9a4e582b9b4b7/Kudu.Core/Scripts/selectNodeVersion.js#L106 for an example of how installed node versions are detected at runtime; maybe we can do something similar for ruby.